### PR TITLE
Issue #1996: merge fasthttp-mysql and fasthttp-postgresql into fasthttp folder

### PR DESCRIPTION
**Pull request by valyala**
This addresses #1996 .

### CODE REVIEW

**Comment by msmith-techempower** [comment]
Something wonky happened in Travis; we need to test this manually.

**Comment by methane** [comment]
I know the issue.
See https://github.com/TechEmpower/FrameworkBenchmarks/pull/2003/commits/68a05ebb3b1bc33f01586f1f6922bcdc6964735f

When benchmark_config is wrong, this error was happen.

**Comment by msmith-techempower** [comment]
Oh interesting, we will have to address that probably. Thanks @methane

File: frameworks/Go/fasthttp/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "fortune_url": "/fortune",
+      "update_url": "/update?queries=",
+      "plaintext_url": "/plaintext",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "Postgresql",
```
Comment (methane) [inline frameworks/Go/fasthttp/benchmark_config.json]: "Postgres"
See here:
https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/benchmark/framework_test.py#L924-L935

Comment (valyala) [inline frameworks/Go/fasthttp/benchmark_config.json]: Fixed

**Comment by nbrady-techempower** [comment]
Hi @valyala we are getting ready to start our preview runs for round-13. In the meantime we are preparing to start work on verifying pull requests for round-14. Because of this, we are slightly restructuring our branches. Would you mind closing this and making a pull request that targets the new round-14 branch? Thanks!

Here's more information: http://frameworkbenchmarks.readthedocs.io/en/latest/Development/Contributing-Guide/#github-pull-request-procedure

**Comment by valyala** [comment]
Sure, I'll re-create the PR for round-14.

**Pull request closed without merging**